### PR TITLE
Updates to shared cleaner functions

### LINKED ISSUE #11: Arbitrary string cleaner
## Context

Would be helpful to have a function that replaces any given string (ideally set in an environment variable, so that sensitive strings can be added) from response and body text.

```python
  clean_strings = os.environ.get('CLEAN_STRINGS', "").split(',')
 for clean_me in clean_strings:
     response['body']['string'] = response['body']['string'].replace(clean_me, 'CLEANED')
```

### PR DESCRIPTION
**Pull request by edthedev**
- Add MyPy support to clean_token
- Fix clean_token to ignore expected 403 auth errors.
- Add clean_env_strings function to clean any strings set in a specific environment variable
- Pin urllib3 to work around VCR.py #668

Closes #11

### CODE REVIEW

**Review by tzturner — DISMISSED** [review]

File: tests/test_env_string.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        }
+    }
+
+
+@pytest.mark.parametrize('body,expected', [
+    ('CLEAN ME', 'CLEANED'),
+    ('CLEAN THIS TOO', 'CLEANED'),
+    ('CLEAN ME and CLEAN THIS TOO', 'CLEANED and CLEANED'),
+    ('NOTHING TO CLEAN', 'NOTHING TO CLEAN'),
+    (b'DEADBEEF', b'DEADBEEF'),
```
Comment (ddriddle) [inline tests/test_env_string.py:22]: You may want to add another test to make it clear that binary strings are not cleaned:

```suggestion
    (b'DEADBEEF', b'DEADBEEF'),
    (b'CLEAN ME', CLEAN ME'),  # Binary strings are not cleaned
```

File: tests/test_jwt_token.py [hunk 2]
```diff
@@ -27,3 +27,13 @@ def test_clean_token():
...
+
+
+def test_clean_env_string():
+    '''
+    Test with a bytes token.
+    Test with a string content.
+    '''
+
+    
+
```
Comment (ddriddle) [inline tests/test_jwt_token.py:39]: Please remove this empty function:

```suggestion
```

**Review by ddriddle — DISMISSED** [review]
Code looks code. I suggested adding a test case and please remove the dead code accidentally added.

File: tests/test_env_string.py (hunk 1 above)
Comment (edthedev) [reply tests/test_env_string.py:22]: Great point. Done.

File: tests/test_jwt_token.py (hunk 2 above)
Comment (edthedev) [reply tests/test_jwt_token.py]: Good catch. Done.

**Review by zdc217 — APPROVED** [review]

**Review by ddriddle — APPROVED** [review]

**Pull request merged**